### ISSUE TITLE
Loss visualization

### ISSUE THREAD

**Issue by AuroraYXL** [megvii-research/LBHomo#3]
I didn't find any places in your code where TensorboardX is used. Is there no loss visualization in your code

**Comment by JianghaiSCU** [comment]
yes, we donot use TensorboardX for  visualization, you can add it by yourself
